test/server/typegen.ts

```diff
@@ -2593,6 +1940,273 @@ test('typegen: typescript w/ postgrestVersion', async () => {
   )
 })
 
+test('typegen: typescript consistent types definitions orders', async () => {
```

Comment:
In there we now have something that should warn us if a regression in generation stability occurs on any of the main entities.